go.mod

```diff
@@ -44,5 +45,8 @@ require (
 	golang.org/x/crypto v0.0.0-20220307211146-efcb8507fb70 // indirect
 	golang.org/x/net v0.0.0-20220225172249-27dd8689420f // indirect
 	golang.org/x/text v0.3.7 // indirect
+	golang.org/x/time v0.0.0-20220224211638-0e9765cccd65 // indirect
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
 )
+
+replace github.com/cloudflare/cloudflare-go => ./deps/cloudflare-go
```

Comment:
This should be replaced by a GitHub fork instead of something local (or preferably changes get upstreamed)